jenkins/Jenkinsfile

```diff
@@ -34,20 +34,24 @@ pipeline {
         container('node'){
           dir("${env.WORKSPACE}/${env.PROJECT}") {
             echo "Install dependencies"
+            sh 'npm i -g npm@8' // seems like with older versions from 8.x branch workspaces doesn't work properly
             sh "chown `whoami` ." // workaround for https://github.com/cypress-io/cypress/issues/20286
             sh 'npm ci --prefer-offline'
 
-            echo "Build application"
-            sh 'npm run build'
+            echo "Build packages"
+            sh 'npm run build --workspaces --if-present'
```

Comment:
`--workspaces` configuration option allows to run npm script in the context of all configured workspaces.
By running the command with the `--if-present` flag, npm will ignore workspaces missing target script.